.gitignore

```diff
@@ -1,6 +1,7 @@
 # Project-specific rules
 autoapi.sqlite
-node_modules/
+node_modules
```

Comment:
Changed this to a non-directory because when using docker-compose, we actually make `node_modules` a symlink. Originally I wanted to have the entrypoint copy `/tmp/node_modules` to the user's `node_modules` directory, but doing this while preserving permissions proved a massive headache with busybox/alpine, so I just made it a symlink.